GNUmakefile

```
@@ -0,0 +1,26 @@
+TEST?=$$(go list ./... |grep -v 'vendor')
```

Using a single static package allows to see the progression of unit tests execution:
```suggestion
TEST?="./provider"
```

See https://github.com/terraform-providers/terraform-provider-helm/commit/7a1cbfaef74a48dcac4704952263782541896d75#diff-b0fa74059a31d3efac99fea6d20ca25aR1